src/libraries/System.Drawing.Common/src/System/Drawing/Graphics.cs

```diff
@@ -708,6 +708,14 @@ public void DrawBezier(Pen pen, Point pt1, Point pt2, Point pt3, Point pt4)
             DrawBezier(pen, pt1.X, pt1.Y, pt2.X, pt2.Y, pt3.X, pt3.Y, pt4.X, pt4.Y);
         }
 
+        /// <summary>
+        /// Draws the outline of a rectangle specified by <paramref name="rect"/>.
+        /// </summary>
```

Comment:
Please also add XML comments for the parameters.

Reply:
I can do that. I originally skipped it because none of other overloads has them.